### DISCUSSION TITLE
Ace Jump

### DISCUSSION THREAD

**Discussion by ElSargo** [helix-editor/helix#3429] [Ideas]
Are there any plans for an Ace Jump like feature to be added? This is something I would love to see in the editor, but might be better suited to a plugin like in IntelliJ or vscode.

**Comment by the-mikedavis** [comment]
It looks like IntelliJ's ace jump is derived from vim's easymotion. There's an issue for clever-f which also covers vim's easymotion to an extent: https://github.com/helix-editor/helix/issues/274
